Add Release Notes for #897 (#1510)
nautobot/nautobot: nautobot/docs/release-notes/version-1.3.md

Python 3.10 is officially supported by Nautobot now, and we are building and publishing Docker images with Python 3.10 now.

#### JSON Type for Custom Fields

> A custom field can now have a type of "json" which will render with a button to toggle this data view in an object's
> detail page in the UI. Simple string searching of these fields is available from the object list view's filter form.

Reviewer: I don't know that these are necessarily important enough points to include in the release overview. Maybe just:

```suggestion
Custom fields can now have a type of "json". Fields of this type can be used to store arbitrary JSON data.
```
